ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/BaseDesktopAppearance.java

```diff
@@ -63,6 +64,10 @@
 
         @Source("org/iplantc/de/theme/base/client/desktop/user.png")
         ImageResource userPrefImg();
+
+        @DataResource.MimeType("font/opentype")
+        @Source("org/iplantc/de/theme/base/client/desktop/Texta_Font/Texta-Bold.otf")
+        DataResource textaBold();
```

Comment:
Good question.  Well, when I remove the definition from `BaseDesktopAppearance` the fonts still look correct and I can switch from Bold to Regular with no issue, so it would seem that it's not necessary to have the fonts defined in both places.  But since I don't fully understand the intricacies of how that all works or if there are random cases where it would stop working without the definition, I'd like to leave it here for now.  I will look into how that all works later though.